### PR TITLE
added constant files and changed gitignore

### PR DESCRIPTION
**Pull request by chirag4601**

### CODE REVIEW

**Review by legolas-1 — COMMENTED** [review]
1. Changes regarding variables are done.
2. Always give one extra line at the end of file.
3. There are still come variables to rename, fix them as you work further.
4. One thing, if you are accessing something from response data, don't assume directly that it's correct and will be present. like `resp['user']`. So use some library that will help in these checks while parsing it,  like lodash in JS. This will protect app from breaking unexpectedly if network call fails etc.
5. always use image at x2 or x3 size.
6. put secrets etc in env file.

File: lib/screens/home.dart [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      child: Scaffold(
+        body: ListView(
+          keyboardDismissBehavior: ScrollViewKeyboardDismissBehavior.onDrag,
+          children: [
+            SizedBox(
+              height: smallSpace,
+            ),
+            LiveasyTitleTextWidget(),
+            SizedBox(
+              height: smallMediumSpace - 2,
```
Comment (chirag4601) [inline lib/screens/home.dart:32]: Want To Ask if doing this contradicts the logic of using a seperate file of spaces as there are few places where random values are used

**Comment by chirag4601** [comment]
1. have a doubt at one point. I have added a comment in the request pls scroll down and find my command
2. will do it
3. sure
4. searched for such package but couldn't find any will look for it again.
5. didn't understood this point. How to use a image in x2 or x3 size if the image is already in size as needed.
6. done. will be updated in next pull

**Comment by liveasy97** [comment]
Also how will I make a constant file for heights and widths as they can be from a very small value like 10 to a large one like 360. How will I generalize such case?

File: lib/screens/home.dart (hunk 1 above)
Comment (legolas-1) [reply lib/screens/home.dart:32]: No, if you have to use some value out of the box use it. But mostly designer make designs such that these values never go out.

**Comment by legolas-1** [comment]
make for like 5px to 60px and leave rest of the heights.. because there will be many different values of height, our main task is to define spaces between widgets so that if we try to open app in tablet or in landscape mode it wont break.

**Comment by legolas-1** [comment]
No worries if there is no lodash like package in flutter.
you can download and save x2 or x3 images and then while using it give it a fixed size or dynamic size according to screen.

**Comment by Atharv-Joshi** [comment]
> 1. Changes regarding variables are done.
> 2. Always give one extra line at the end of file.
> 3. There are still come variables to rename, fix them as you work further.
> 4. One thing, if you are accessing something from response data, don't assume directly that it's correct and will be present. like `resp['user']`. So use some library that will help in these checks while parsing it,  like lodash in JS. This will protect app from breaking unexpectedly if network call fails etc.
> 5. always use image at x2 or x3 size.
> 6. put secrets etc in env file.

4 .  like null safety?

**Review by liveasy97 — APPROVED** [review]

**Pull request merged**